app/controllers/posts_controller.rb

```diff
@@ -6,7 +6,33 @@ def index
 
   def show
     @post = Post.find(params[:id])
-    @user = @post.author
+    @user = User.find(params[:user_id])
+    @comment = Comment.new
     @comments = @post.comments
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
+    @post.author = current_user
+    if @post.save
+      redirect_to user_post_path(current_user, @post)
+    else
+      render :new
+    end
+  end
+
+  # def comment
+  #   @post = Post.find(params[:id])
+  #   @comment = Comments.new(post: @post.id, author: current_user.id,  text: params[:text])
+  #   @comment.save
+  #   redirect_to user_post_path(current_user.id, Post.find(params[:post_id]))
+  # end
```

Comment:
- [ ] It would be better not to commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

Reply:
Well resolved 👍🏽